uui/components/datePickers/RangeDatePicker.tsx

```diff
@@ -19,6 +19,10 @@ export interface RangeDatePickerProps extends BaseRangeDatePickerProps, SizeMod
      * A pure function that gets placeholder for 'from' or 'to' input.
      */
     getPlaceholder?(type: InputType): string;
+    /**
+    * Defines id for inner input
+    */
+    id?: string;
```

Comment:
Let's change to -
/** HTML ID attribute for the first input into toggler */

Reply:
fixed